.github/workflows/canary_images.yaml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
       - main
+      - "image/**"
+      - "*image"
+      - "image*"
```

Comment:
@mumoshu I would like to try docker build on some branch without pushing to registry